flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/AggregateTableOperation.java

```diff
@@ -53,6 +53,12 @@ public TableSchema getTableSchema() {
 		return tableSchema;
 	}
 
+	@Override
+	public String asSummaryString() {
+		return String.format("Aggregate: (group: %s, agg: %s)", groupingExpressions, aggregateExpressions) +
+			TableOperationUtils.indent(this.child.asSummaryString());
```

Comment:
nit: We could reduce code duplication if `String.format` and calling the children is done in `TableOperation`?

Reply:
I thought about that, but for that it would be best to make the `TableOperation` an abstract class, which I was unsure if we should do, but I think it is ok. I updated it.